src/views/Simulation/utils/SceneContainer.js

```diff
@@ -5,21 +5,22 @@ import { Point, Container } from 'pixi.js';
 const MIN_ZOOM = 1;
 const MAX_ZOOM = 8;
 const ZOOM_SPEED = 0.1;
+const ORIGIN = { x: 0, y: 0 };
 
 const interpolateLinear = (a, b, t) => {
   return a + (b - a) * t;
 };
 
 export class SceneContainer extends Container {
-  constructor(app, containerRef) {
+  constructor(sceneApp, canvasSceneRef) {
     super();
     this.pivot.x = this.width / 2;
     this.pivot.y = this.height / 2;
 
     this.zoom = 1;
     this.tickerAttached = false;
```

Comment:
If `this.tickerAttached` is no longer used, shouldn't we remove this line?

Reply:
I thought I removed it. Maybe a undo-redo or reset --hard miss.